cmd/issue/create-issue.go

```diff
@@ -42,6 +42,9 @@ func createIssue(options *issueCommandOptions) (*jira.Issue, error) {
 			Description: options.description,
 			Project:     *project,
 			Type:        *issueType,
+			Watches:     &jira.Issue.Watches{
```

Comment:
I was following the wrong package: https://github.com/andygrunwald/go-jira/blob/main/cloud/issue.go#L115

Let me check the correct one.